R/auth_sitrep.R

```diff
@@ -0,0 +1,187 @@
+#' Twitter Tokens sitrep
+#' 
+#' Get a situation report of your current tokens; useful for upgrading from 
+#' rtweet 0.7.0 to 1.0.0 and diagnosing problems with tokens. Called by it side effects
+#' 
+#' Searches for old tokens on the user folder, if duplicate tokens are found 
+#' they are deleted. Tokens are then moved to the new location. 
+#' @return Invisible the path of the tokens
+#' @export
+#' @examples
+#' auth_sitrep()
+auth_sitrep <- function() {
+  all_tokens_files <- c(find_old_tokens(), 
+                        find_rappdirs_tokens(), 
+                        find_tools_tokens())
+  path <- tools::R_user_dir("rtweet", "config")
+  if (!dir.exists(path)) {
+    dir.create(path, showWarnings = FALSE, recursive = TRUE)
+  }
```

Comment:
It's very hard to review this with all these codecov comments. Is there someway to suppress them?

Reply:
I wasn't even aware that this happened. I don't know when did they start appearing, but it is from the test-coverage workflow... I haven't found any way to change that (but haven't looked into the configuration of the workflow). Sorry!

Reply:
I [found the way](https://docs.codecov.com/docs/common-recipe-list#set-non-blocking-status-checks,), I need to change codecov.yml to use informational: false. Will do that in this PR and on the other branches

Reply:
Great, thanks!